api/application.go

```diff
@@ -184,6 +169,7 @@ func (h ApplicationHandler) Update(ctx *gin.Context) {
 	m.ID = id
 	db := h.DB.Model(m)
 	db = db.Omit(clause.Associations)
+	db = db.Omit("Bucket")
```

Comment:
The read-only permission removed from the model so we need to enforce not updating the bucket ref in the API.